layouts/partials/fragments/single-member.html

```diff
@@ -0,0 +1,134 @@
+{{ $single := .self.Params }}
+{{ $name := .self.Name }}
+{{ $name_t := strings.TrimSuffix ".md" $name }}
+{{ $member := .resources.Match (printf "%s/*" $name_t) }}
+{{ "<!-- Single Member -->" | safeHTML }}
+{{ $bg := $single.background | default "light"}}
+
+<section {{ with $single.self_link }}id="{{ . }}"{{ end }}>
+  <div class="overlay container-fluid
+    {{- printf " bg-%s" $bg -}}
+  ">
+    <div class="container d-flex flex-column py-3">
+      {{ with $single.title }}
+        <div class="row">
+          <div class="col text-center
+            {{- if or (eq $bg "white") (eq $bg "light") (eq $bg "secondary") (eq $bg "primary") -}}
+              {{- printf " text-%s" "dark" -}}
+            {{- else -}}
+              {{- printf " text-%s" "light" -}}
+            {{- end -}}
+          ">
+            <h2>
+              {{- . | markdownify -}}
+            </h2>
+          </div>
+        </div>
+      {{ end }}
+      <!-- {{ with $single.subtitle }}
+        <div class="row">
+          <div class="col text-center pt-2 pb-3
+            {{- if or (eq $bg "secondary") (eq $bg "primary") -}}
+              {{- printf " text-%s" "dark" -}}
+            {{- else -}}
+              {{- printf " text-muted text-%s" "secondary" -}}
+            {{- end -}}
+          ">
+            <h5>
+              {{- . | markdownify -}}
+            </h5>
+          </div>
+        </div>
+      {{ end }} -->
+      <div class="card py-1 mt-5">
+        <div class="row">
+          {{ range $member }}
+            {{ $member := .Params }}
+            <div class="row flex-sm-columns">
+              <div class="col-lg-5 col-md-6 col-sm-12 pt-3 text-center text-sm-center">
+                <div class="col d-flex">
+                  <div class="col-lg-12 col-md-11 col-sm-12 ml-lg-5 text-sm-center
+                    {{- if or (eq $bg "white") (eq $bg "light") (eq $bg "secondary") (eq $bg "primary") -}}
+                      {{- printf " text-%s" "dark" -}}
+                    {{- else -}}
+                      {{- printf " text-%s" "light" -}}
+                    {{- end -}}
+                  ">
+                    <img src="/images/member/{{ $member.image }}" class="img-fluid rounded-circle w-75 border
+                    {{- if or (eq $bg "white") (eq $bg "light") (eq $bg "secondary") (eq $bg "primary") -}}
+                      {{- printf " border-%s" "dark" -}}
+                    {{- else -}}
+                      {{- printf " border-%s" "white" -}}
+                    {{- end -}}
+                    "alt=""></img>
```

Comment:
Since we have the member title it would be nice if that is shown here for better accessibility.

Reply:
Also I think `<img />` is more suitable since img tag does not accept any child nodes.

Reply:
Yeah true about the image tag must have missed it also the title is displayed or you mean before the image tag?

Reply:
I mean with the `alt` attribute of `img` tag. That's used for screen readers or slow internet connections to show a title for the image in case the image is not yet loaded.